samples/bot-calling-meeting/csharp/Bots/CallingBot.cs

```diff
@@ -92,46 +86,6 @@ public CallingBot(BotOptions options, IConfiguration configuration, ICard card,
             }
         }
 
-        protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersAdded, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
I noticed that this was being fired on install or after being added to a chat. I've removed this to prevent me having to deal with it. Instead the default message response from the bot will return the welcome card which will guide users in the useage of the app